app/views/ReportView/components/Immune/index.tsx

```diff
@@ -10,6 +10,15 @@ import withLoading, { WithLoadingInjectedProps } from '@/hoc/WithLoading';
 import { hlaColumnDefs, cellTypesColumnDefs } from './columnDefs';
 import { ImmuneType, HlaType } from './types';
 
+import './index.scss';
+
+const IMAGE_KEYS = [
+  'cibersort.combined_t-cell_scatter',
+  'cibersort.cd8_positive_t-cell_scatter',
+  'mixcr.circos_trb_vj_gene_usage',
+  'mixcr.dominiance_vs_alpha_beta_t-cells_scatter',
```

Comment:
is this the right mixcr name? dominiance, not dominance?

Reply:
fixed